tests/test_prep_run_nvnmd_train.py

```
@@ -0,0 +1,517 @@
+):
+    with open(fname) as fp:
+        lines_ = fp.read().strip().split("\n")
+    if only_check_name:
+        lines = []
+        for ii in lines_:
+            ww = ii.split(" ")
+            ww[1] = str(Path(ww[1]).name)
+            lines.append(" ".join(ww))
+    else:
+        lines = lines_
+    revised_fname = lambda ff: Path(ff).name if only_check_name else Path(ff)
```

Ruff (0.11.9)

Do not assign a `lambda` expression, use a `def`

Rewrite `revised_fname` as a `def`

(E731)